### ISSUE TITLE
SELKS 4 Repository Key missing

### ISSUE THREAD

**Issue by TomStageDK** [StamusNetworks/SELKS#119]
Hi,

Apparently the key used to sign the SELKS 4 Repository is no loger in place.

I am trying to update my newly installed SELKS 4 system and I get this message from apt-get update:
root@suricata:~# apt-get update
Ign:1 http://packages.stamus-networks.com/selks4/debian stretch InRelease
Hit:2 http://packages.elastic.co/curator/4/debian stable InRelease
Ign:3 http://packages.stamus-networks.com/selks4/debian-kernel stretch InRelease
Ign:5 https://artifacts.elastic.co/packages/5.x/apt stable InRelease
Hit:6 https://artifacts.elastic.co/packages/5.x/apt stable Release
Get:7 http://packages.stamus-networks.com/selks4/debian stretch Release [991 B]
Ign:4 http://evebox.org/files/debian stable InRelease
Get:9 http://packages.stamus-networks.com/selks4/debian-kernel stretch Release [991 B]
Ign:11 http://packages.stamus-networks.com/selks4/debian stretch Release.gpg
Get:10 http://evebox.org/files/debian stable Release [821 B]
Ign:12 http://packages.stamus-networks.com/selks4/debian-kernel stretch Release.gpg
Ign:13 http://evebox.org/files/debian stable Release.gpg
Reading package lists... Done
E: The repository 'http://packages.stamus-networks.com/selks4/debian stretch Release' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://packages.stamus-networks.com/selks4/debian-kernel stretch Release' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://files.evebox.org/evebox/debian stable Release' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

Looking at https://github.com/StamusNetworks/SELKS/wiki/Using-the-SELKS-Debian-Repositories it states that the key can be installed from http://packages.stamus-networks.com/packages.selks4.stamus-networks.com.gpg.key, but running the wget command says that there is no valid key.

**Comment by pevma** [comment]
Could you share a bit more details of what exactly are the steps to reproduce that?
I just tried myself but could not reproduce - 
```
root@SELKS:~# apt-get update 
Hit:1 http://security.debian.org/debian-security stretch/updates InRelease
Ign:2 http://debian.lth.se/debian stretch InRelease                                                                                                                                                               
Hit:3 http://debian.lth.se/debian stretch-updates InRelease                                                                                                                                                       
Hit:4 http://debian.lth.se/debian stretch Release                                                                                                                                                                 
Hit:6 http://packages.stamus-networks.com/selks4/debian stretch InRelease                                                                                                                        
Hit:7 http://packages.elastic.co/curator/4/debian stable InRelease      
Hit:9 http://packages.stamus-networks.com/selks4/debian-kernel stretch InRelease                
Hit:5 http://evebox.org/files/debian stable InRelease                                           
Hit:10 http://packages.stamus-networks.com/selks4/debian-test stretch InRelease
Ign:11 https://artifacts.elastic.co/packages/5.x/apt stable InRelease
Hit:13 https://artifacts.elastic.co/packages/5.x/apt stable Release
Reading package lists... Done
root@SELKS:~# 

```

**Comment by TomStageDK** [comment]
Hi,

I am not sure, but apparently I have an issue with my newly installed system, I am still debugging.

But inj essence I am not able to download from any repository on my system, I tried to install Webmin per this page: http://www.webmin.com/deb.html
But when i try to download the Key it fails, but I can download the Key on my workstation and import it on the Newly installed SELKS server, but it complains when I run the apt-get update command with an error similay to one above.

I have tried to run apt-get clean with no luck.
